libr/socket/socket_http_server.c

```diff
@@ -105,6 +105,11 @@ R_API RSocketHTTPRequest *r_socket_http_accept (RSocket *s, RSocketHTTPOptions *
 	}
 	if (content_length>0) {
 		r_socket_read_block (hr->s, (ut8*)buf, 1); // one missing byte wtf
+		if (ST32_ADD_OVFCHK (content_length, 1)) {
```

Comment:
I think you should create a new `INT_ADD_OVFCHK`, because it is not said that int == st32 everywhere, AFAIK.

Reply:
look at the file libr/include/r_types_base.h
`11 #define st32 int"

I made the same mistake

Reply:
ah mmm :D ok!